base_report_assembler/__openerp__.py

```diff
@@ -34,11 +35,13 @@
 To install this assemblage option for specific object you can install
 the folling module(s):
 
-- Invoices: invoice_report_assemble (lp:account-invoice-report)
+- Assemble invoice report: `invoice_report_assemble`
+(git@github.com:OCA/account-invoice-report)
 
 """,
  'author': "Camptocamp,Odoo Community Association (OCA)",
  'maintainer': 'Camptocamp',
+ 'license': 'AGPL-3',
  'website': 'http://www.camptocamp.com/',
```

Comment:
Seems website field is a mess

grep -rn website --include=__openerp__.py | grep -v OCB | cut -d ':' -f 4- | cut -d "'" -f 2 | cut -d '"' -f 2 | sort | uniq -c | sort -nr
    159 http://www.camptocamp.com
    114 http://www.savoirfairelinux.com
     54 http://www.agilebg.com
     49 
     38 http://www.akretion.com
     36 http://camptocamp.com
     35 http://www.akretion.com/
     34 http://miketelahun.wordpress.com
     27 http://www.julius.fr
     24 http://www.camptocamp.com/
     20 https://github.com/YannickB/community-management
     14 http://www.openerp.com
     12 http://www.openerp-italia.org
     12 http://www.acsone.eu
     12 http://launchpad.net/oemedical
     11 http://openerpbrasil.org
     10 https://github.com/OCA/bank-payment
      9 http://openerp.camptocamp.com
      9 http://acsone.eu